homepage.js

```diff
@@ -24,11 +24,13 @@
      * @param events: JSON event object fetched from Bedeworks
      */
     var initializeEventsHomepage = function(eventsJson) {
-        var allEvents = CTLEventsManager.loadEvents(eventsJson);
-        jQuery('#homepage-calendar-listing').append(renderEvents(allEvents, ITEMS_TO_DISPLAY));
+        var today = new Date();
+        var allHomepageEvents = CTLEventsManager.loadEvents(eventsJson, today);
+        jQuery('#homepage-calendar-listing').append(renderEvents(allHomepageEvents, ITEMS_TO_DISPLAY));
     };
 
     $(document).ready(function() {
+        $('head').append('<style type="text/css"> .event{margin-top: 2em;} .event:first-child {margin-top: 0em;}');
```

Comment:
Yeah, in this case I didn't want to add the overhead of another call on the home page for such a small amount of css.  The other option would be to add it to Divi's custom css editor, but then it wouldn't be committed or portable with the widget.